vector/src/main/java/im/vector/app/features/home/room/list/RoomListFragment.kt

```diff
@@ -373,6 +380,18 @@ class RoomListFragment @Inject constructor(
         }
     }
 
+    private fun listenItemCount(section: RoomsSection, sectionAdapter: SectionHeaderAdapter) {
```

Comment:
Nit: I would rename to `listenToItemCount`, or `observeItemCount` (Rx style). Not sure what is the Flow style naming convention. `collectItemCount`?

Reply:
Okay, I will go for `observeItemCount` to be agnostic of the used observation framework (could be LiveData, Flow, Rx, etc.).